### ISSUE TITLE
raco pkg install fails on Windows due to certificate error for html-parsing dependency

### ISSUE THREAD

**Issue by greghendershott** [dstorrs/racket-dstorrs-libs#2]
```
raco.exe pkg install html-parsing
Resolving "html-parsing" via https://download.racket-lang.org/releases/7.1/catalog/
Resolving "html-parsing" via https://pkgs.racket-lang.org
Downloading http://www.neilvandyke.org/racket/html-parsing.zip
ssl-connect: connect failed (error:1416F086:SSL routines:tls_process_server_certificate:certificate verify failed)
```

0. I only have one Windows machine at hand, so I'm not 100% sure this will be true for all Windows users. However other packages hosted on e.g. GitHub download fine.

1. This problems happens trying both at command-line and in DrRacket package manager GUI.

2. Reporting here because `html-parsing` isn't on any of the repo hosting sites with issues-reporting features.

3. As it happens, I don't need `HTML-Element`. Someday it would be nice if handy were split into separate packages with finer dependencies.

**Comment by greghendershott** [comment]
Also `html-parsing` transitively requires `mcfly` and `overeasy` which have the same problem.

It is possible to install manually:

- Use a web browser to download each of `https://www.neilvandyke.org/racket/{mcfly overeasy html-parsing}.zip`.

- Go to a command prompt, cd to `Downloads`, then doing three `raco pkg install {mcfly overeasy html-parsing}.zip` (must do `html-parsing` last).

But that's not good for CI, and very not good for end users.

**Comment by dstorrs** [comment]
Yeah, this is a new thing that I only noticed in the last few days.  I
think something may have broken upstream, because I didn't use to have
these issues.

And yes, handy definitely needs to be split up.

**Comment by dstorrs** [comment]
handy is stable and has been for years.  It's not going anywhere.  At some point I intend to take its submodules, minimize their dependencies (especially on each other) and push them out as separate libraries so that if you want to use handy/list-utils you don't also need to pull in handy/HTML-Element and handy/web

It has extensive documentation, but it's in the form of comments in the code.  I'll pull it into the README for now so that it shows up on Github and then Scribblify it when I have energy.
